repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -1247,13 +1247,14 @@ private Object mapToVertexByTypeCategory(AttributeMutationContext ctx, EntityMut
                     deleteDelegate.getHandler().deleteEdgeReference(currentEdge, ctx.getAttrType().getTypeCategory(), ctx.getAttribute().isOwnedRef(),
                             true, ctx.getAttribute().getRelationshipEdgeDirection(), ctx.getReferringVertex());
                 }
+                if(StringUtils.isNotEmpty(edgeLabel) && Objects.nonNull(newEdge)) {
```

Comment:
StringUtils.isNotEmpty() checks for both null & empty, no need to add Objects.nonNull()